examples/nextjs/src/components/my-page.js

```diff
@@ -40,6 +41,17 @@ const componentsMap = {
     CustomNoComponent: CustomNoComponent,
 };
 
+// Init Configuration
+DotCmsClient.init({
+    dotcmsUrl: process.env.NEXT_PUBLIC_DOTCMS_HOST,
+    authToken: process.env.NEXT_PUBLIC_DOTCMS_AUTH_TOKEN,
+    siteId: "59bb8831-6706-4589-9ca0-ff74016e02b2",
+    requestOptions: {
+        // In production you might want to deal with this differently
+        cache: "no-cache",
+    }
+});
```

Comment:
No no, no super no... we need to fix this. :)

Reply:
I'll remove this, I was testing something. This is not the ticket to fix this

Reply:
This is the ticket for that: https://github.com/dotCMS/core/issues/29640